app/controllers/api/v1/users_controller.rb

```diff
@@ -38,14 +32,19 @@ def destroy
     @user.destroy
     render json: { message: "User deleted successfully!" }
   end
-end
 
-private
+  # Add profile action and pass current_user as params
+  def profile
+    render json: UserProfileSerializer.new(current_user, { params: { current_user: current_user } }).serializable_hash
+  end
+
+  private
 
-def set_user
+  def set_user
   @user = User.find(params[:id])
-end
+  end
 
-def user_params
-  params.require(:user).permit(:first_name, :last_name, :role)
+  def user_params
+  params.require(:user).permit(:first_name, :last_name)
+  end
```

Comment:
No idea why the linter didn't error on this
```suggestion
  def user_params
    params.require(:user).permit(:first_name, :last_name)
  end
```